pkg/client/testclient/fake.go

```diff
@@ -9,19 +9,10 @@ import (
 	"github.com/openshift/origin/pkg/client"
 )
 
-// FakeAction is a wrapper around the Kubernetes FakeAction.
-// Used for faking various actions on resources.
-type FakeAction ktestclient.FakeAction
-
 // Fake implements Interface. Meant to be embedded into a struct to get a default
 // implementation. This makes faking out just the method you want to test easier.
 type Fake struct {
-	// Fake by default keeps a simple list of the methods that have been called.
-	Actions []FakeAction
-	Err     error
-	// ReactFn is an optional function that will be invoked with the provided action
-	// and return a response.
-	ReactFn ktestclient.ReactionFunc
+	*ktestclient.Fake
```

Comment:
Followed that example but I had to make changes upstream. I needed to make the client's Actions slice a pointer (as in useridentitymapping tests) or it cannot share its content otherwise.